Mathlib/Data/Set/Pointwise/Basic.lean

```diff
@@ -718,6 +718,37 @@ theorem div_union : s / (t₁ ∪ t₂) = s / t₁ ∪ s / t₂ :=
 #align set.div_union Set.div_union
 #align set.sub_union Set.sub_union
 
+lemma union_div_trans {α : Type} (a b c : Set α) (ha : b ⊆ a) (hb : c ⊆ b) :
```

Comment:
This lemma should have `diff`, not `div` in the name.

Reply:
It also seems to me that it likely doesn't belong in this file. This file is for pointwise operations on sets, but this lemma is about set differences.

Reply:
In fact, it seems this lemma already exists as [Set.diff_union_diff_cancel](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Set/Basic.html#Set.diff_union_diff_cancel)